### ISSUE TITLE
Suggestion: More consistent ore yield

### ISSUE THREAD

**Issue by sinopike** [GregTechCE/GregTech#490]
Currently 5 banded iron dust can be electrolysed into 2 iron dust and 3000mB oxygen. However each banded iron dust can also be smelted into one iron ingot.
This created an inconsistency, as whether should one unit of banded iron dust contains one unit of iron or 2/5 unit?
Can this inconsistency be solved like it is in Gregtech 6? Where impure ores like malachite, galena, cassiterite produce nuggets when smelted?

**Comment by Archengius** [comment]
no, not really. forcibly nerfing direct smelting is not a way to fix that balance issue. Instead, it will make ore useless.

**Comment by sinopike** [comment]
Then how about buff electrolyser to produce one iron dust and 1500mB oxygen per banded iron dust? At least give people a reason to use this receipt, and stay consistent.

**Comment by Archengius** [comment]
you cannot "buff" electrolyzer, it breaks mass conversion rules.

**Comment by sinopike** [comment]
But smelting is breaking the rule already, if you can't get one iron out of one banded iron from electrolyser then why can you do so from furnace? I am not trying to nerf ores, just to remove this buff.

**Comment by Archengius** [comment]
smelting is not breaking that rule, by smelting banded iron you get iron alloy.

**Comment by sinopike** [comment]
smelting banded iron in a vanilla furnace gives iron ingot, which equals one iron dust. I have just checked again to make sure.
5 banded iron ==electrolyse==> 2 iron + 3 oxygen
5 banded iron ==furnace====> 5 iron
furnace can't turn oxygen into iron, right?
Other than banded iron, I have also found magnesite, penlandite, cobaltite, malachite, cassiterite and probably a few other ores to have this problem too.

However the blast furnace recipe for magnetite, galena does produce nuggets at a correct proportion, and they are certainly not nerfing. Can we just make other ores yield like them?
